samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -0,0 +1,163 @@
+in the Zephyr documentation.
+
+Antenna matrix configuration
+============================
+
+To use this sample with Angle of Departure enabled, additional GPIOS configuration
+is required to enable control of the antenna array. Example of such configuration
+is provided in :file:`nrf52833dk_nrf52833.overlay`.
+
+:file:`nrf52833dk_nrf52833.overlay` is a device tree overlay that provides information
+about which GPIOS should be used by Radio peripheral to switch between antenna
+patches during CTE transmission in AoD mode. At least two GPIOs must be provided
+to enable antenna switching. GPIOS are used by Radio peripheral in order following
+indices of :code:`dfegpio#-gpios` properties. The order is important because it
+affects mapping of antenna switch patterns to GPIOS (see `Antenna patterns`_).
+
+To successfully use Direction Finding Beacon with AoD mode enabled provide
+following data related with owned antenna matrix design:
+
+* provide GPIO pins to :code:`dfegpio#-gpios` propreties in :file:`nrf52833dk_nrf52833.overlay` file
+* provide default antenna that will be used to transmit PDU :code:`dfe-pdu-ant`
+  property in :file:`nrf52833dk_nrf52833.overlay` file
+* update antenna patterns in :cpp:var:`ant_patterns` array in :file:`main.c`.
+
+Antenna patterns
+================
+
+The antenna switch pattern is a binary number where each bit is applied to
+a particular antenna GPIO pin. For example, the pattern 0x3 means that antenna
+GPIOs at index 0,1 will be set, and following are left unset.
+
+This also means that, for example, when using four GPIOs, the patterns count
+cannot be greater than 16 and maximum allowed value is 15.
+
+If the number of switch-sample periods is greater than the number of stored
+switch patterns, then the radio loops back first pattern.
+
+The following table presents the patterns that you can use to switch antennas on
+the Nordic-designed antenna matrix:
+
++--------+--------------+
+|Antenna | PATTERN[3:0] |
++========+==============+
+| ANT_12 |  0 (0b0000)  |
++--------+--------------+
+| ANT_10 |  1 (0b0001)  |
++--------+--------------+
+| ANT_11 |  2 (0b0010)  |
++--------+--------------+
+| RFU    |  3 (0b0011)  |
++--------+--------------+
+| ANT_3  |  4 (0b0100)  |
++--------+--------------+
+| ANT_1  |  5 (0b0101)  |
++--------+--------------+
+| ANT_2  |  6 (0b0110)  |
++--------+--------------+
+| RFU    |  7 (0b0111)  |
++--------+--------------+
+| ANT_6  |  8 (0b1000)  |
++--------+--------------+
+| ANT_4  |  9 (0b1001)  |
++--------+--------------+
+| ANT_5  | 10 (0b1010)  |
++--------+--------------+
+| RFU    | 11 (0b1011)  |
++--------+--------------+
+| ANT_9  | 12 (0b1100)  |
++--------+--------------+
+| ANT_7  | 13 (0b1101)  |
++--------+--------------+
+| ANT_8  | 14 (0b1110)  |
++--------+--------------+
+| RFU    | 15 (0b1111)  |
++--------+--------------+
+
+Dependencies
+************
```

Comment:
Are there any dependencies?
If not, this can be removed?

Reply:
Oh, I see that this belongs to the text further down. It can be moved.

Reply:
moved to bottom of the file, where it belongs.